### PR TITLE
[WIP-f] 7 - Add abstract BaseTweetHandler and coding examples

### PR DESCRIPTION
**Pull request by tklarryonline**
#7 - Add an abstract `BaseTweetHandler` so that developers can subclass it and create their own handlers

- [x] Create `BaseTweetHandler`
- [x] Create `TweepyBaseTweetHandler` extends `BaseTweetHandler` for anyone who wants to develop using `tweepy`
- [ ] Write examples code and add it into `readme.md` for developers to follow.

### CODE REVIEW

**Comment by tklarryonline** [comment]
@anhhuy1605, @giang271291 and @hieueastagile: Please take a look. This PR is quite messy.

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,6 @@ var/
 
 # Testing
 .coverage
+
+# Project's settings
+env.sh
```
Comment (hieueastagile) [inline .gitignore]: Oh, I think it's better that we could consider this https://github.com/mattseymour/python-env instead? It follow the current standard that everyone is using.

Comment (tklarryonline) [inline .gitignore]: Nice idea! I was curious about how Heroku handles `.env` file, but now you got me :wink:

Comment (tklarryonline) [inline .gitignore]: @hieueastagile Switch to using it in ac87fba. +1 for you :+1:

**Pull request merged**